tools/windows/DatadogAgentInstaller/CustomActions/UserCustomActions.cs

```diff
@@ -42,22 +61,13 @@ public static string GetRandomPassword(int length)
         /// </remarks>
         private static string GetDefaultDomainPart()
         {
-            try
-            {
-                var serverInfo = NetServerGetInfo<SERVER_INFO_101>();
-                if ((serverInfo.Type & ServerTypes.DomainCtrl) == ServerTypes.DomainCtrl
-                    || (serverInfo.Type & ServerTypes.BackupDomainCtrl) == ServerTypes.BackupDomainCtrl)
-                {
-                    // Computer is a DC, default to domain name
-                    var joinedDomain = Domain.GetComputerDomain();
-                    return joinedDomain.Name;
-                }
-                // Computer is not a DC, default to machine name
-            }
-            catch (ActiveDirectoryObjectNotFoundException)
+            if (HostIsDomainController())
```

Comment:
Would not it be useful to log is host is domain controller (its domain name is used)